apps/documentation/src/routes/documentation/tutorial/components.mdx

```diff
@@ -127,3 +128,10 @@
   );
 }
 ```
+
+import NavigationButtons from '../../../components/navigation-buttons'
+
+<NavigationButtons 
+		prev={{title: 'API fetching', href:"/documentation/tutorial/api-fetching"}}
+		next={{title: 'Dynamic routes', href:"/documentation/tutorial/dynamic-routes"}}
```

Comment:
**[LanguageTool]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unpaired symbol: ‘"’ seems to be missing (EN_UNPAIRED_QUOTES)
URL: https://languagetool.org/insights/post/punctuation-guide/#what-are-parentheses 
Rule: https://community.languagetool.org/rule/show/EN_UNPAIRED_QUOTES?lang=en-US
Category: PUNCTUATION
<!-- __reviewdog__:ChAyNTRiMTc1MzA3NjAxYmMxEgxMYW5ndWFnZVRvb2w= -->